### DISCUSSION TITLE
Maven Central upload

### DISCUSSION THREAD

**Discussion by neinoi** [sngular/scs-multiapi-plugin#324] [Q&A] [answered]
Hi,

This build has sent the 5.3.4 version to Maven Central : https://github.com/sngular/scs-multiapi-plugin/actions/runs/7832651800/job/21371857100

Unfortunately, two days later the plugin is still not visible on https://s01.oss.sonatype.org/#nexus-search;gav~com.sngular~scs-multiapi-maven-plugin~~~~kw,versionexpand

What’s going on ? Any idea ?

**Comment by jemacineiras** [comment, marked as answer]
Hi,
upload to Maven Central require to a manual "close & release" procedure, just didn´t remember that. 
My apologies. I just released now. It should be available now.
Cheers

**Reply by neinoi** [reply]
I saw it 5 minutes ago, thanks !
